### ISSUE TITLE
mermaid.js contains HTML and not the mermaid javascript

### ISSUE THREAD

**Issue by danhooper** [glureau/dokka-mermaid#2]
As of release 0.4.3 the mermaid.js file contains:

```
<html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>nginx</center>
</body>
</html>
```

**Comment by glureau** [comment]
Damn I'm sorry about that issue, I guess a bad deployment and I didn't upgrade the plugin on my old project. Thanks for noticing and reporting it. I've just published the 0.4.4 that fixes this issue.

**Comment by danhooper** [comment]
Thanks!
